src/adagrams.js

```diff
@@ -1,15 +1,161 @@
+  const LETTER_POOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+const SCORE_CHART = {
+    A: 1, 
+    B: 3, 
+    C: 3, 
+    D: 2, 
+    E: 1, 
+    F: 4, 
+    G: 2, 
+    H: 4, 
+    I: 1, 
+    J: 8, 
+    K: 5, 
+    L: 1, 
+    M: 3, 
+    N: 1, 
+    O: 1, 
+    P: 3, 
+    Q: 10, 
+    R: 1, 
+    S: 1, 
+    T: 1, 
+    U: 1, 
+    V: 4, 
+    W: 4, 
+    X: 8, 
+    Y: 4, 
+    Z: 10
+}
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+
+  let letters = {...LETTER_POOL}
+  let letterCount = Object.keys(letters).reduce(((total, letter) => total += letters[letter]), 0);
+
+  let letterArr = [];
+
+  for (let i = 0; i < 10; i++) {
```

Comment:
10 is an int literal hanging out in your code, or a [magic number](https://dev.to/ruben_alapont/magic-numbers-and-magic-strings-its-time-to-talk-about-it-ci2). Prefer we use a variable to reference this value to make your code more readable and self-documenting.

```suggestion
  const numCardsInHand = 10;
  for (let i = 0; i < numCardsInHand; i++) {
```